docs/source/tutorial/interfaces.rst

```diff
@@ -62,9 +62,8 @@ going to use it to convert each element to a ``String``:
             show' (x :: Nil) = show x
             show' (x :: xs)  = show x ++ ", " ++ show' xs
 
-Note that we need the explicit ``forall n .`` in the ``show'`` function
-because otherwise the ``n`` is already in scope, and fixed to the value of
-the top level ``n``.
+Note that we need the explicit ``forall n .`` in the ``show'`` function;
+otherwise, ``n`` would refer to the top-level ``n`` already in scope.
```

Comment:
Potentially, one could add a comma after "function", but otherwise I find that the text reads better as it was originally.

Reply:
Done.